src/app/js/formats/streamgraph/utils.js

```diff
@@ -177,28 +177,9 @@ export function getMinMaxValue(stackedData) {
     return { minValue, maxValue };
 }
 
-export function cutStr(str, column_number) {
-    const strSplit = str.split(/-| /);
-    let resStr = '';
-    let tokenCounter = 0;
-
-    let maxLength;
-    if (column_number == 3) {
-        maxLength = 31;
-    } else maxLength = 31;
-
-    while (
-        tokenCounter < strSplit.length &&
-        `${resStr} ${strSplit[tokenCounter]}`.length < maxLength
-    ) {
-        resStr = `${resStr} ${strSplit[tokenCounter]}`;
-        tokenCounter++;
-    }
-
-    if (tokenCounter < strSplit.length) {
-        resStr = `${resStr} [...]`;
-    }
-    return resStr;
+export function cutStr(str, maxLegendLength) {
+    const slicedStr = str.slice(0, maxLegendLength);
```

Comment:
Perhaps clearer with [`substring`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring) rather than `slice`?

Reply:
Definitively clearer.